packages/kbn-securitysolution-autocomplete/src/field_value_wildcard/index.tsx

```diff
@@ -0,0 +1,245 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React, { useCallback, useMemo, useState, useEffect } from 'react';
+import { EuiFormRow, EuiComboBoxOptionOption, EuiComboBox } from '@elastic/eui';
+import { DataViewBase, DataViewFieldBase } from '@kbn/es-query';
+
+import { uniq } from 'lodash';
+
+import { ListOperatorTypeEnum as OperatorTypeEnum } from '@kbn/securitysolution-io-ts-list-types';
+
+// TODO: I have to use any here for now, but once this is available below, we should use the correct types, https://github.com/elastic/kibana/issues/100715
+// import { AutocompleteStart } from '../../../../../../../src/plugins/data/public';
+type AutocompleteStart = any;
+
+import * as i18n from '../translations';
+import { useFieldValueAutocomplete } from '../hooks/use_field_value_autocomplete';
+import {
+  getGenericComboBoxProps,
+  GetGenericComboBoxPropsReturn,
+} from '../get_generic_combo_box_props';
+import { paramIsValid } from '../param_is_valid';
+
+const SINGLE_SELECTION = { asPlainText: true };
```

Comment:
nit/opinion/no changes requested...

You can do either of those to help enforce immutability. To let you know, I more and more lean towards just use functions to enforce immutability such as something like this (warning I did not test this, this is pseudo code):
```ts
const getSingleSelection = () => ({ asPlainText: true });
```

Reasons being:
* If you have accidental circular module issues the functions will be less of an issue if in most/all cases a non-issue with circular deps.
* No one can turn "off" my TypeScript to mutate what the function returns. They have to actually change the function to be a non-function.
* If the function/code path is never called it does not consume in-memory resources. This will consume in-memory resources as soon as the module is imported since it's holding data and will stick around until the module is unloaded.
* You can pass the function if you really wanted to and evaluate lazy later on.
* It enforces the immutability at the runtime level so I don't have to worry about recursive freeze, seal, etc... if i grow the object.

We aren't as strict about this as I have seen on other systems. But there are some systems in the past I have seen where everything was to be defined as a function rather than constants and after a while it seemed better than using modules.